scripts/upgrade

```diff
@@ -28,6 +28,9 @@ db_name=$(ynh_app_setting_get --app=$app --key=db_name)
 db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
 random_key=$(ynh_app_setting_get --app=$app --key=random_key)
 email=$(ynh_user_get_info --username=$admin --key=mail)
+dav_support=$(ynh_app_setting_get --app=$app --key=dav_support)
+signup=$(ynh_app_setting_get --app=$app --key=signup)
+two_factor=$(ynh_app_setting_get --app=$app --key=two_factor)
```

Comment:
Those new variables need to be created for upgrade from previous versions

Reply:
Added the key values to be created while upgrade.